<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>40</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Drone Corridor Manager&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerNew">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Register New Corridor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listExisting">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>List Existing Corridors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adminLogin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Approve New Corridor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mapExisting">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Map Of Corridors</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
